common/data_refinery_common/utils.py

```diff
@@ -262,6 +262,13 @@ def calculate_sha1(absolute_file_path):
         for buf in iter(partial(open_file.read, io.DEFAULT_BUFFER_SIZE), b""):
             hash_object.update(buf)
```

Comment:
It looks like you stole the line `return hash_object.hex_digest()` from the end of this function